content/en/docs/kubernetes/operator/automatic.md

```diff
@@ -310,6 +322,39 @@ in the previous step.
 > Operator you **MUST** set these env variables to `http/protobuf`, or python
 > auto-instrumentation will not work.
 
+#### Auto-instrumenting Python logs
+
+By default, Python logs auto-instrumentation is disabled. If you would like to
+enable this feature, you must to set the `OTEL_LOGS_EXPORTER` and
+`OTEL_PYTHON_LOGGING_AUTO_INSTRUMENTATION_ENABLED` environment variables as
+follows:
+
+```yaml
+apiVersion: opentelemetry.io/v1alpha1
+kind: Instrumentation
+metadata:
+  name: python-instrumentation
+  namespace: application
+spec:
+  exporter:
+    endpoint: http://demo-collector:4318
+  env:
+  propagators:
+    - tracecontext
+    - baggage
+  python:
+    env:
+      - name: OTEL_LOGS_EXPORTER
+        value: otlp_proto_http
+      - name: OTEL_PYTHON_LOGGING_AUTO_INSTRUMENTATION_ENABLED
+        value: 'true'
+```
+
+> Note that `OTEL_LOGS_EXPORTER` must be explicitly set to `otlp_proto_http`,
```

Comment:
@avillela a PR in the operator to automatically default the logs exporting behavior to http/proto like we do for traces and metrics would be awesome.